CONTRIBUTING.md

```diff
@@ -12,15 +12,15 @@ For contributing to this repository follow following steps:
 ## Point to look for
 1. When submitting PR:
     * Make sure your PR adds only one functionality or try to fix one issue per PR.
-    * Don't try to do a lot in one PR i.e. stick to scope of your PR.
+    * Don't try to do a lot in one PR i.e. stick to the scope of your PR.
     * Your PR should take code from one stable state to another stable state.
+
+2. For Commits:
+    * Try to have proper commit messages which tell what you commit dos.
     * If your PR solves some issue, refer that issue in your PR.
     * Provide proper description what your PR do.(Description in terms of Points will be plus).
     * If your are still working on PR. Label it (Work in Progress) and Add TODO's in description (How to do that given at end of doc) and keep marking once done.
```

Comment:
Would Change do following changes here ?
1. s/you/your/
2. s/description/the description/